packages/docs/src/components/page-configs/styles/colors/page-config.ts

```diff
@@ -0,0 +1,39 @@
+import { ApiDocsBlock } from '@/types/configTypes'
+import { DocsHelper } from '@/helpers/DocsHelper'
+
+const scheme = `
```

Comment:
That'll be pretty confusing for end user.

Instead we can have a structure like this.
* Overview
* Setup
  * Usage example
  * Default colors
* In-app usage
  * CSS helpers + example
  * How to use CSS variables
  * How to override CSS variables